javascript-source/discord/commands/customCommands.js

```diff
@@ -454,10 +454,18 @@
                 return;
             }
 
-            $.inidb.set('discordChannelcom', action, subAction);
             $.discord.clearChannelCommands(action);
-            $.discord.setCommandChannel(action, subAction, false);
-            $.discord.say(channel, $.discord.userPrefix(mention) + $.lang.get('discord.customcommands.channelcom.success', action, subAction));
+            var key = [];
```

Comment:
This was not made to be automated, it took me a while to get the logic down, i'm afraid what changing it might cause other issues later on.

Reply:
Without this changes, it's currently impossible to !channelcom command to more than one channel. It can be global or used only in one text channel. I don't know about future changes, but if you think it may cause troubles, I can cancel this commit. But then it shouldn't be said "spectate the channels with a comma and space for multiple" here https://phantombot.tv/commands/discord/ in documentation.

Reply:
The command works as documented, at least it did when I tested the last time.

Reply:
It can't work properly because when you try to add several channels like this
!channelcom somecommand chan1, chan2, chan3
it adds only the first argument "subAction = args[1];", "chan1," (with comma) in DB $.inidb.set('discordChannelcom', action, subAction); and via discord function $.discord.setCommandChannel(action, subAction, false);

Reply:
You need to use quotes, I forgot to mention that in the documentation. `!channelcom "chan1, chan2, chan3"`

Reply:
So quotes do the trick then :) ok

Reply:
We can simply join the args too, that's another possibility.